client/lib/plans/features-list.js

```diff
@@ -587,6 +587,18 @@ export const FEATURES_LIST = {
 			i18n.translate( 'Upload more images, videos, audio, and documents to your website.' ),
 	},
 
+	[ constants.FEATURE_200GB_STORAGE ]: {
+		getSlug: () => constants.FEATURE_200GB_STORAGE,
+		getTitle: () =>
+			i18n.translate( '{{strong}}200 GB{{/strong}} Storage Space', {
```

Comment:
@travisw How should we test this? Are the strings already translated elsewhere?

Reply:
Yep, this should already be translated. It's pulled from here:
https://github.com/Automattic/wp-calypso/blob/448191032ddf297ff5803cdc6648b11c73307511/client/lib/plans/features-list.js#L288